store/tikv/2pc.go

```diff
@@ -1058,10 +1058,12 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 
 	commitDetail := c.getDetail()
 	commitDetail.PrewriteTime = time.Since(start)
-	// TODO(youjiali1995): Record the backoff time of the last finished batch. It doesn't make sense to aggregate all batches'.
 	if bo.GetTotalSleep() > 0 {
-		atomic.AddInt64(&commitDetail.CommitBackoffTime, int64(bo.GetTotalSleep())*int64(time.Millisecond))
+		boSleep := int64(bo.GetTotalSleep()) * int64(time.Millisecond)
```

Comment:
It's only useful if there is only one batch and it will add more BackoffTypes if it backoffs when locating keys. The simplest solution is to move the L1067 to the `if` block.